### PR TITLE
Calendar

### PR DESCRIPTION
**Pull request by ovkulkarni**

### CODE REVIEW

File: intranet/apps/schedule/views.py [hunk 1]
```diff
@@ -149,30 +148,39 @@ def week_view(request, date=None, render_template=True):
...
 
-# does NOT require login
+
+@login_required
+def calendar_view(request):
+    data = {}
+    data["week_data"] = week_data(request)
+    data["month_data"] = month_data(request)
+    if 'view' in request.GET and request.GET["view"] == "month":
+            data["view"] = "month"
```
Comment (tjhsst-ion) [inline intranet/apps/schedule/views.py]: Codacy Issue found: [Bad indentation. Found 12 spaces, expected 8](https://www.codacy.com/app/tjcsl/ion/file/2769759962/issues/source?bid=3176164&fileBranchId=3362832#l178)

Comment (tjhsst-ion) [inline intranet/apps/schedule/views.py]: Codacy Issue found: [Bad indentation. Found 12 spaces, expected 8](https://www.codacy.com/app/tjcsl/ion/file/2770047317/issues/source?bid=3176164&fileBranchId=3362832#l178)

**Comment by pefoley2** [comment]
@ovkulkarni Please add a test :)

**Comment by pefoley2** [comment]
Ping.
@ovkulkarni: What's the status of this?

**Comment by pefoley2** [comment]
@ovkulkarni ping

**Comment by pefoley2** [comment]
@ovkulkarni is this still relevant? please rebase if so.

**Comment by pefoley2** [comment]
@ovkulkarni Now rebased, ready for merge?

**Comment by ovkulkarni** [comment]
@pefoley2 yeah its ready

**Pull request merged**